content/html-css/blocks/what-is-html/index.md

```diff
@@ -0,0 +1,101 @@
+  <header>
+    <h1>Title</h1>
+  </header>
+</article>
+```
+
+</details>
+<details>
+<summary>What are some examples of web browsers?</summary>
+
+Chrome, Firefox, Safari, Lynx, JAWS, Brave, Edge, Vivaldi, Opera...
+
+</details>
+<details>
+<summary>Here’s an interesting question: what does HTML produce in the browser?</summary>
+
+It produces what we see, of course. But in a little more detail: the browser **parses** the HTML we write, executes the linked Javascript, loads the CSS, media, and other resources, and produces a _model_ of our document, called the document object model. It produces an **API** called the **DOM**.
+
+</details>
+
+<details>
+<summary>What is parsing?</summary>
+
+Make sense of. Analyse **syntactically**.
+
+</details>
+
+<details>
+<summary>What is syntax?</summary>
+
+The rules that structure language, so it can be understood by someone else. I put words in an order, according to rules, I _structure meaning_ and you can get that meaning back out if you know the same rules. Programming languages are the same - they each have a set of rules, they each have a _syntax_.
+
+</details>
+
+<details>
+<summary>What is an API?</summary>
+
+API stands for Application Programming Interface. Imagine you are at a train station: You ask the station master what trains are running today, and they read you a list. You ask when the next train to Manchester is, and they tell you it has been delayed.
+
+The station master has _lots more_ information on the trains, but when you ask your question, they know precisely what information you want, based on your shared understanding of the question. The DOM is the station master; the DOM is the interface.
+
+</details>
+
+<details>
+<summary>What is semantic HTML? What does semantic mean?</summary>
+
+Semantics means _meaning_. Semantic HTML is _meaningful code_: each piece of data marked up with correct, that is to say, descriptive tags. A heading has an h1 tag. A button has a button tag. There are only two html tags that are deliberately NULL, or have no meaning.
+
+These tags are then _interpreted_ by APIs to give functionality to the page. A button tag is interpreted by the browser as a button, and so it can be clicked, or triggered in many ways. A heading tag is interpreted by the browser as a heading, and so provides a traversible page outline to a screen reader.
+
+HTML tags are powerful because they are meaningful. They are meaningful because they are semantic. Semantic HTML is powerful HTML.
+
+</details>
+
+<details>
+<summary>Why is semantic HTML important?</summary>
+
+HTML tags are powerful because they are meaningful. They are meaningful because they are semantic. Semantic HTML is powerful HTML.
+
+The better structured your document, the more meaning you can pack into it, the more powerful your code is, the better it will work in more contexts, and more things will be able to interface with it. The more meaning you put in, the more meaning other readers can get out.
+
```

Comment:
So, I didn't put this because it's not persuasive, even though it's practically gospel at this point. But IME I find it counterproductive. When new developers read this they dismiss it because they think all they need to do is to attach the information _they want the user to have_ to a div. They don't know what they don't know. 

Instead I want them to encounter the idea in class in the devtools workshop, and then develop it along with their API understanding. Can you look at that part and see where you'd expand (bearing in mind we want to limit our verbiage).

Reply:
That assumes we can control what they know about HTML already or what they see around them. ;) But we cannot, so they are all swimming in div soup, trust me.